<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Houdini_Child_renamer</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>170</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Parent Node</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rename_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>150</height>
     </size>
    </property>
    <property name="text">
     <string>Rename all Childs</string>
    </property>
   </widget>
   <widget class="QListWidget" name="child_listWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>281</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="parent_comboBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="refresh_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>230</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>150</height>
     </size>
    </property>
    <property name="text">
     <string>Refresh_list</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
